src/pages/portfolio/Portfolio.tsx

```diff
@@ -42,6 +43,9 @@ export default () => {
   const { freeCollateral } = useSelector(getSubaccount, shallowEqual) || {};
   const { nativeTokenBalance } = useAccountBalance();
 
+  const numPositions = useSelector(getExistingOpenPositions)?.length || 0;
```

Comment:
When using `useSelector`, if the return is an array or object, you need to add `shallowEqual` as the second param.

Reply:
If you wanted, you could add a new selector function that returns the length so that we only trigger a re-render if the `numPositions` changes instead of when `existingOpenPositions` changes (which happens whenever the market data gets updated)

Reply:
ah that makes sense - will do!